### PR TITLE
[cli] Add a big warning to stop people from modifying the global cli

### PR DESCRIPTION
**Pull request by vjeux**

### CODE REVIEW

File: react-native-cli/index.js [hunk 1]
```diff
@@ -1,9 +1,38 @@
...
+// /!\ DO NOT MODIFY THIS FILE /!\
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+//
+// react-native-cli is installed globally on people's computers. This means
+// that it is extremely difficult to have them upgrade the version and
+// because there's only one global version installed, very prone to breaking
+// changes.
+//
+// The only job of react-native-cli is to init the repository and then
+// forwards all the commands to the local version of react-native.
```
Comment (mkonicek) [inline react-native-cli/index.js]: nit: forward

Comment (vjeux) [inline react-native-cli/index.js]: (y)

File: react-native-cli/index.js [hunk 2]
```diff
@@ -1,9 +1,38 @@
...
+ * of patent rights can be found in the PATENTS file in the same directory.
  */
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+// /!\ DO NOT MODIFY THIS FILE /!\
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+//
+// react-native-cli is installed globally on people's computers. This means
+// that it is extremely difficult to have them upgrade the version and
+// because there's only one global version installed, very prone to breaking
```
Comment (mkonicek) [inline react-native-cli/index.js]: nit: it is very prone to

File: react-native-cli/index.js [hunk 3]
```diff
@@ -1,9 +1,38 @@
...
+// changes.
+//
+// The only job of react-native-cli is to init the repository and then
+// forwards all the commands to the local version of react-native.
+//
+// If you need to add a new command, please add it to local-cli/.
```
Comment (mkonicek) [inline react-native-cli/index.js]: :+1:

File: react-native-cli/index.js [hunk 4]
```diff
@@ -1,9 +1,38 @@
...
+//
+// The only reason to modify this file is to add more warnings and
+// troubleshooting information for the `react-native init` command.
+//
+// Do not make breaking changes! We absolutely don't want to have to
+// tell people to update their global version of react-native-cli.
```
Comment (mkonicek) [inline react-native-cli/index.js]: This paragraph repeats what has been said above but fine if you want to make this message super duper scary :)

**Pull request merged**